### ISSUE TITLE
Sharing your training loss curve

### ISSUE THREAD

**Issue by tuan3w** [G-Wang/WaveRNN-Pytorch#3]
Hi @G-Wang,
I'm developing a [cnn based vocoder](https://github.com/tuan3w/cnn_vocoder). And I want to try beta_mle_loss as used in your code. However, as I test, training wavernn is too slow. Can you share with me your training loss curve, so I can have a good reference when I test it to my model. 
Thanks.

**Comment by G-Wang** [comment]
Hi @tuan3w , I didn't log my loss curve in my implementation yet, as I only save checkpoints. I'll be sure to add loss plot to the repo asap.

Beta distribution, and gaussian distribution in general seems to be very finicky to learn. You can hear from my audio samples that the voice has weird intonations, and those samples were when the network's loss has stopped decreasing for a while. The other gaussian distribution training on github takes a very long time as well (https://github.com/Rayhane-mamah/Tacotron-2/issues/173#issuecomment-432068628.)

In terms of training the beta distribution that I had the checkpoint for download, as it was some of my initial tries, I trained with fixed learning rate of 1e-4, and decayed it by half when I see training stalls. (final learning rate was very small, probably 1e-8). The average loss per epoch towards the end is around -6.3.

I've updated the repo recently with learning rate schedules that's been tested (somewhat), I'll update the beta distribution training with loss curves once they're done and post it here.

**Comment by tuan3w** [comment]
Thanks for your information.
